documentation/features/case/zgw/zgw-documents/access-control.md

```
@@ -4,7 +4,7 @@
 Available since Valtimo `12.10.0`.
 {% endhint %}
 
-Access to the documents that were uploaded to the case, can be configured through access control. More information about access control can be found [here](https://docs.valtimo.nl/features/access-control).
+Access to the documents that were uploaded to the case, can be configured through access control. More information about access control can be found [here](../../../access-control/README.md).
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)